tasks/radosbench.py

```diff
@@ -59,12 +61,13 @@ def task(ctx, config):
             profile_name = None
 
         pool = 'data'
-        if config.get('pool'):
-            pool = config.get('pool')
-            if pool is not 'data':
-                ctx.manager.create_pool(pool, erasure_code_profile_name=profile_name)
-        else:
-            pool = ctx.manager.create_pool_with_unique_name(erasure_code_profile_name=profile_name)
+        if config.get('create_pool', True):
+            if config.get('pool'):
+                pool = config.get('pool')
+                if pool is not 'data':
```

Comment:
nitpick: I would prefer != here as we're comparing values not identities.  This should work fine though.